Libraries/LibWeb/CodeGenerators/Generate_CSS_PropertyID_cpp/Generate_CSS_PropertyID_cpp.cpp

```diff
@@ -59,15 +59,17 @@ int main(int argc, char** argv)
     ASSERT(json.is_object());
 
     out() << "#include <AK/Assertions.h>";
+    out() << "#include <AK/String.h>";
     out() << "#include <LibWeb/CSS/PropertyID.h>";
     out() << "namespace Web {";
     out() << "namespace CSS {";
 
-    out() << "PropertyID property_id_from_string(const StringView& string) {";
+    out() << "PropertyID property_id_from_string(const StringView& view) {";
+    out() << "    auto string_lowercase = view.to_string().to_lowercase();";
 
     json.as_object().for_each_member([&](auto& name, auto& value) {
         ASSERT(value.is_object());
-        out() << "    if (string == \"" << name << "\")";
+        out() << "    if (string_lowercase == \"" << name << "\")";
```

Comment:
Can we use `string.equals_ignoring_case()` here instead of converting the input to lowercase?